lang/en.json

```diff
@@ -28,7 +28,7 @@
         "Filter": "LDAP Search Filter",
         "FilterDescription": "An LDAP filter used to determine which entities are allowed to be used for authentication, e.g. \"(objectClass=person)\".",
         "Kerberos": "Use Web Server Auth (e.g. Kerberos SSO)",
-        "KerberosDescription": "If enabled, the plugin will check the $_SERVER['REMOTE_USER'] variable and assume the user is already authenticated. Normal super users are still allowed to authenticate normally.",
+        "KerberosDescription": "If enabled, the plugin will check the $_SERVER['REMOTE_USER'] variable and assume the user is already authenticated. All users are still allowed to authenticate normally.",
```

Comment:
maybe replace the whole string by `If enabled, the plugin will check the $_SERVER['REMOTE_USER'] variable and assume the user is already authenticated.`. by not specifying the other string, it should imply that it fallsback to the other mechanism (which may depend on whether the `Always use LDAP for authentication` is checked.